pkg/util/multierror/multierror.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package multierror
+
+import (
+	"fmt"
+)
+
+// derived from https://github.com/golang/appengine/blob/master/errors.go
+
+// MultiError is returned by batch operations.
+type MultiError []error
```

Comment:
we already have `AggregateError`, what's the difference? seems no need to define multi error here.

Reply:
I didn't know about Aggregate. I updated the PR to use the Aggregate API. Thanks!